File: Linguagem Dart/1008.dart
import 'dart:io';

void main()
{
    int numeroFuncionario = int.parse(stdin.readLineSync());
    double horaTrabalhada = double.parse(stdin.readLineSync());
    double valorHoraTrabalhada = double.parse(stdin.readLineSync());
    
    double Salario = horaTrabalhada * valorHoraTrabalhada;
    
    print("NUMBER = $numeroFuncionario");
    print("SALARY = U\$ " + Salario.toStringAsFixed(2));
}

File: Liguagem C#/1008.cs
using System; 

class URI {

    static void Main(string[] args) { 
        
    int numeroFuncionario = int.Parse(Console.ReadLine());
    double horasTrabalhada = double.Parse(Console.ReadLine());
    double valorHoraTrabalhada = double.Parse(Console.ReadLine());
    
    double salario = horasTrabalhada * valorHoraTrabalhada;
    
    Console.Write("NUMBER = {0}\n",numeroFuncionario);
    Console.Write("SALARY = U$ {0:F2}\n",salario);

    

    }

}
